tests/baselines/reference/ambientModuleDeclarationWithReservedIdentifierInDottedPath.errors.txt

```diff
@@ -1,27 +1,27 @@
 ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,1): error TS2304: Cannot find name 'declare'.
-ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,9): error TS2580: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
-ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,16): error TS2819: Namespace name cannot be 'debugger'.
-ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,25): error TS1005: ';' expected.
+ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,9): error TS2304: Cannot find name 'namespace'.
+ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,19): error TS2819: Namespace name cannot be 'debugger'.
+ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts(11,28): error TS1005: ';' expected.
 
 
 ==== ambientModuleDeclarationWithReservedIdentifierInDottedPath.ts (4 errors) ====
     // https://github.com/microsoft/TypeScript/issues/7840
     
-    declare module chrome.debugger {
+    declare namespace chrome.debugger {

```

Comment:
I wonder if they still use `module`?

Reply:
Searching github shows no results other than this test.